src/Leaflet.Renderer.SVG.Tile.js

```diff
@@ -39,6 +39,16 @@ L.SVG.Tile = L.SVG.extend({
 		}
 	},
 
+	removeFrom: function (map) {
+		if (this.options.interactive) {
+			for (var i in this._layers) {
+				var layer = this._layers[i];
+				delete this._map._targets[L.stamp(layer._path)];
```

Comment:
Of course you're right, I meant the `Layer` subclasses that `FeatureLayer` is mixed into.

Unfortunately, working out which code paths are »safe« to take is quite hairy, mostly due to the way the `Renderer` subclasses have been repurposed and the fact that `PolylineLayer` and siblings are at best pseudo-layers…